cg/meta/demultiplex/validation.py

```diff
@@ -83,8 +83,10 @@ def validate_flow_cell_delivery_status(flow_cell_output_directory: Path, force:
 
 
 def is_flow_cell_ready_for_postprocessing(
```

Comment:
Feels a bit weird to me that this is a void function rather than a boolean given the naming. Also makes the test assertions a bit weird - `assert is_flow_cell_ready_for_postprocessing is None` feels closer to `assert not is_flow_cell_ready_for_postprocessing` than the opposite.

Reply:
agree

Reply:
I agree with you both. Let's keep that for another refactoring. I think it will require a bit too much change right now.